src/handler.rs

```diff
@@ -220,6 +234,21 @@ pub struct ActorResponse<A, I, E> {
     item: ActorResponseTypeItem<A, I, E>,
 }
 
+impl<A, I, E> fmt::Debug for ActorResponse<A, I, E> {
+    fn fmt(&self, fmt: &mut fmt::Formatter) -> fmt::Result {
+        let item = format!(
+            "ActorResponseTypeItem::{}",
+            match self.item {
+                ActorResponseTypeItem::Result(_) => "Result(_)",
+                ActorResponseTypeItem::Fut(_) => "Fut(_)",
+            }
+        );
+        fmt.debug_struct("ActorResponse")
```

Comment:
I think you can put formatting into match self.item instead and avoid allocating temporary item